crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -1634,6 +1635,7 @@ impl OlmMachine {
     /// println!("{:?}", device);
     /// # });
     /// ```
+    #[instrument()]
```

Comment:
If you really don't need to pass any arguments to `instrument`, you can leave out the inner parentheses.